packages/conf-openblas/conf-openblas.0.2.0/files/centos_install.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+yes | yum update
+yes | yum install git
+yes | yum install gcc gcc-c++ gcc-gfortran make kernel-devel
+
+git clone https://github.com/xianyi/OpenBLAS.git
+cd OpenBLAS
+make && make install
```

Comment:
Does it require sudo for the `make install` step?

Reply:
TBH no idea, this is left as-is from the 0.1 version.
/cc @ryanrhymes who wrote this originally...

Reply:
See my previous comment. I think this file is obsolete now, you can find all the dependency in [CentOS dockerfile](https://github.com/owlbarn/owl/blob/master/docker/Dockerfile.centos)

Similarly, all the dependency on various distributions can be found in [this folder](https://github.com/owlbarn/owl/tree/master/docker)

Reply:
Nice work with those `Docker` files!! Thanks for pointing them out